From: Jon Bernard <jbernard@debian.org>
Date: Wed, 15 Dec 2010 09:53:04 -0500
Subject: Fix Provides LSB keyword in cgred initscript
To: libcg-devel@lists.sourceforge.net

This change is required for Debian compatibility. Here is the relevant
lintian information:

    This /etc/init.d script indicates it provides one or more facilities,
    but none of the provided facilities match the name of the init script.
    In certain cases, it may be necessary to not follow that convention, but
    normally init scripts should always provide a facility matching the name
    of the init script.

    Refer to http://wiki.debian.org/LSBInitScripts for details.

I am not sure if this change is compatible with other distributions.
This is something that needs to be looked into.  Ideally, we could
settle on a change that works for all parties.
---
 scripts/init.d/cgred.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/scripts/init.d/cgred.in b/scripts/init.d/cgred.in
index 09b2c1e..d23472c 100644
--- a/scripts/init.d/cgred.in
+++ b/scripts/init.d/cgred.in
@@ -22,9 +22,9 @@
 # pidfile: /var/run/cgred.pid
 #
 ### BEGIN INIT INFO
-# Provides:		cgrulesengd
 # Required-Start:	$local_fs $syslog $cgconfig
 # Required-Stop:	$local_fs $syslog
+# Provides:		cgred
 # Default-Start:        2 3 4 5
 # Default-Stop:         0 1 6
 # Short-Description:	start and stop the cgroups rules engine daemon
-- 
